.hourglass/build.yaml

```
@@ -1,4 +1,4 @@
 container:
-  registry: "" # optional, defaults to dockerhub
+  registry: "" # optional, defaults to ghcr
```

nit: I think we should remove this. We aren't defaulting to anything so sort-of misleading.

---

👍